Assets/Scripts/Models/Character.cs

```diff
@@ -357,6 +366,13 @@ public void WriteXml(XmlWriter writer)
         }
 
         writer.WriteAttributeString("needs", needString);
+
+        StringBuilder statsString = new StringBuilder();
+        foreach (KeyValuePair<string, Stat> stat in stats)
+        {
+            statsString.Append(stat.Value.statType).Append(";").Append(stat.Value.Value).Append(":");
+        }
+        writer.WriteAttributeString("stats", statsString.ToString());
```

Comment:
Oh. I didn't saw that the needs where done in that way. The needs code has several issues that need fixing. You shouldn't change the Needs in this PR since this PR is about Stats, but we should fix them in another PR.